image-layout.md

```diff
@@ -5,7 +5,7 @@ This layout MAY be used in a variety of different transport mechanisms: archive
 
 Given an image layout and a ref, a tool can create an [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md) by:
 
-* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [manifest list](manifest-list.md#manifest-list)
+* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [image index](image-index.md#manifest-list)
```

Comment:
> OCI Image Directory Storage

WTAF. The image layout is __NOT__ directory storage. OCI image spec _DOES NOT_ prescribe a specific directory layout for storing images.

Stop propagating your fantasy. It is confusing people and holding things up.